spec/outcome_spec.rb

```diff
@@ -0,0 +1,61 @@
+describe Outcome do
```

Comment:
What I like about your test is the descriptive variable names. I understand what you're testing and why.

Reply:
yay! :) I'll take use this as an example to compare my other tests